js/ui/map.js

```diff
@@ -202,6 +203,11 @@ var Map = module.exports = function(options) {
     this.resize();
 
     if (options.classes) this.setClasses(options.classes);
+    if (options.sourceTypes) {
```

Comment:
Should we name it `customSourceTypes` to reflect that we supply those additionally to the core ones?